setup.py

```diff
@@ -46,9 +48,6 @@
     'pyfr.solvers.euler.kernels',
     'pyfr.solvers.euler.kernels.bcs',
     'pyfr.solvers.euler.kernels.rsolvers',
-    'pyfr.solvers.navstokes',
```

Comment:
This looks like it will strip out the Navier-Stokes solver from the distribution.